wetlab/views.py

```diff
@@ -2142,55 +2142,25 @@ def record_samples(request):
                 "wetlab/record_sample.html",
                 {"fields_info": fields_info},
             )
-        else:
-            # validate mandatory and redundant samples
-            validation = core.utils.samples.validate_sample_data(
-                excel_json_data, req_user, __package__
-            )
-
-            for val in validation:
```

Comment:
Here you can change the for, for the `validation_result`, but I would keep the remaining try/catch, I use those in case there is some error in the data processing during save, it should only arise in case there is a code error I know, but it's a safety measure